app/src/main/share-service-context.js

```diff
@@ -29,14 +29,10 @@ export default class ShareServiceContext {
   }
 
   notify(text) {
-    // TODO: Add Kap icon as `icon` option
-    // TODO: Waiting for: https://github.com/electron/electron/pull/9269
-    // (new electron.Notification({title, body})).show();
-
-    electron.app.kap.mainWindow.send('show-notification', {
+    (new electron.Notification({
```

Comment:
Time to add the `icon`? https://github.com/electron/electron/blob/master/docs/api/notification.md#new-notificationoptions-experimental